src/Tools/Source/RunTests/ProcDumpUtil.cs

```diff
@@ -63,7 +63,8 @@ internal static Process AttachProcDump(string procDumpFilePath, int processId, s
             // /accepteula command line option to automatically accept the Sysinternals license agreement.
             // -ma	Write a 'Full' dump file. Includes All the Image, Mapped and Private memory.
             // -e	Write a dump when the process encounters an unhandled exception. Include the 1 to create dump on first chance exceptions.
-            const string procDumpSwitches = "/accepteula -ma -e";
+            // -f C00000FD.STACK_OVERFLOWC Dump when a stack overflow first chance exception is encountered. 
+            const string procDumpSwitches = "/accepteula -ma -e -f C00000FD.STACK_OVERFLOW";
```

Comment:
Would we also be able to add System.Reflection.TargetParameterCountException to troubleshoot the DoEvents hangs we've been seeing?

Reply:
Do you know how to express that as `-f` switch? Happy to add if we can figure that out.